extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -230,7 +238,7 @@ const connectToolsToMeasurementService = (servicesManager: AppTypes.ServicesMana
         annotationToMeasurement(toolName, annotationAddedEventDetail);
       }
     } catch (error) {
-      console.warn('Failed to update measurement:', error);
+      console.error('Failed to update measurement:', error);
```

Comment:
This had better not be error unless you ensure that annotationToMeasurement is actually found/doesn't throw an exception when the measurement is simply not registered.

Reply:
It's catching an error, no?

Reply:
It is catching an error, but it is one that happens at least semi-regularly.  I agree it SHOULD be an error, but that requires fixing the code to handle a few situations that aren't well handled.  You can convert it, but then you need to go through the various unhandled conversion types and make sure that they don't throw errors any longer, but just warn with a message about unsupported conversion types.  That is, the flow should be that all normal situations are handled without exceptions, and only if the code is incorrect should it throw an exception.

Reply:
Updated.